src/i18n/he.json

```diff
@@ -239,20 +247,12 @@
     "ko": "한국어",
     "nlBE": "Dutch (Belgium)",
     "ro": "Romanian",
-    "svSE": "Swedish (Sweden)",
-    "he": "עברית"
-  },
-  "time": {
-    "unit": "Time Unit",
-    "seconds": "Seconds",
-    "minutes": "Minutes",
-    "hours": "Hours",
-    "days": "Days"
+    "svSE": "Swedish (Sweden)"
   },
   "role": {
-    "input": "הרשאה",
-    "read": "קריאה",
-    "owner": "בעלים"
+    "input": "Role",
+    "read": "Read",
+    "owner": "Owner"
```

Comment:
also here,
I stopped commenting about translations at some point, I think it's worth to recheck what is in use and what's not...

Reply:
@omerzamir  User-Service wasn’t working while I was translating and adding support to rtl so I didn’t change anything related to it for now

Reply:
OK, So ill keep it open until we'll know...